index.bs

```diff
@@ -3183,10 +3183,15 @@ associated with or [=scoped=] to, respectively.
                 on {{PublicKeyCredentialEntity/name}}'s value prior to displaying the value to the user or
                 including the value as a parameter of the [=authenticatorMakeCredential=] operation.
 
-          - When inherited by {{PublicKeyCredentialUserEntity}}, it is a [=human palatability|human-palatable=] identifier for a
-            [=user account=]. It is intended only for display, i.e., aiding the user in determining the difference between user
-            accounts with similar {{PublicKeyCredentialUserEntity/displayName}}s. For example, "alexm", "alex.mueller@example.com"
-            or "+14255551234".
+          - When inherited by {{PublicKeyCredentialUserEntity}}, it is a
+            [=human palatability|human-palatable=] identifier for a [=user account=]. This
+            identifier is the primary value displayed to users by [=Clients=] to help users
+            understand with which [=user account=] a credential is associated. The [=[RP]=] MAY
+            extend the identifier to help a user differentiate between multiple credentials for
+            a single [=user account=] when the same value would otherwise be used for all
+            credentials.
```

Comment:
Sorry, I think this still misses the mark a bit. One authenticator should almost never have two credentials for the same account, but may have credentials for different accounts.

Perhaps we're getting hung up on what an "account" is. You've mentioned that what you're trying to reflect here is a "multi-tenant" use case, where you have one user login account that contains multiple distinct domains of privileges, and uses different WebAuthn credentials to access each domain, but all under the same "umbrella" account. For example, a management console where one user manages several distinct application domains. I think the idea was that all of the user's domain-specific WebAuthn credentials would have the same `user.name`, for example the e-mail address used as a username to log in to the single "umbrella" account that manages all of those domains. Correct?

But within the context of WebAuthn, the term "[user account](https://w3c.github.io/webauthn/#user-account)" is defined as:

>a [user account](https://w3c.github.io/webauthn/#user-account) denotes the mapping of a set of [credentials](https://w3c.github.io/webappsec-credential-management/#concept-credential) [[CREDENTIAL-MANAGEMENT-1]](https://w3c.github.io/webauthn/#biblio-credential-management-1) to a (sub)set of a [Relying Party](https://w3c.github.io/webauthn/#relying-party)'s resources, as maintained and authorized by the [Relying Party](https://w3c.github.io/webauthn/#relying-party).

A WebAuthn credential doesn't need to be tied to an e-mail address, for example. It's perfectly fine for the "multi-tenant" application to associate a WebAuthn credential with a particular privilege domain instead of the "umbrella" account. The distinction here is that each of those privilege domains is a separate "user account" by the WebAuthn definition - they are not "sub-accounts" of the "umbrella" account, because the "umbrella" account is not a WebAuthn "user account" if no WebAuthn credentials are associated with it.

Does that make sense?

Reply:
If so, perhaps it could suffice to just extend the examples?

```
          - When inherited by {{PublicKeyCredentialUserEntity}}, it is a
            [=human palatability|human-palatable=] identifier for a [=user account=]. This
            identifier is the primary value displayed to users by [=Clients=] to help users
            understand with which [=user account=] a credential is associated.

            Examples of suitable values for this identifier include, "alexm", "+14255551234",
            "alex.mueller@example.com", "alex.mueller@example.com (ACME Corporation)",
            or "alex.mueller@example.com (ОАО Примертех)".

```

Reply:
> You've mentioned that what you're trying to reflect here is a "multi-tenant" use case, where you have one user login account that contains multiple distinct domains of privileges, and uses different WebAuthn credentials to access each domain, but all under the same "umbrella" account. For example, a management console where one user manages several distinct application domains.

This exactly. For most RP's `user.name` will be as simple as an email address or username. But for a cloud hosting PaaS, Alex could be the admin of multiple customers' VPS' so the PaaS RP should append some unique value per VPS to his PaaS username when they allow Alex to register authenticators for logging into those admin panels using WebAuthn. 

I've incorporated your suggestions and tweaked the examples accordingly. Let me know what you think.

Reply:
@emlun Can you please take a look at this latest change and see if it satisfies your concern?